lib/rack/ecg/check/redis_connection.rb

```diff
@@ -3,17 +3,26 @@ module Rack
   class ECG
     module Check
       # @!method initialize
-      #   Checks whether the global Redis client is currently connected to the
-      #   database.
+      #   Checks whether the given Redis client is currently connected to the
+      #   database as identified by the ++instance++ option.
       #
-      #   Does not take any options.
+      # @option parameters instance [Redis,Hash] Redis parameters to check
       class RedisConnection
+        attr_reader :instance_parameters
```

Comment:
Consider renaming this to something more accurate. E.g., `redis_instance`?

Reply:
The `sequel` check uses `connection_parameters` to indicate that it takes the necessary parameters to _construct_ a sequel database instance — I'm wondering if we should take the same approach here.

Reply:
Given the possible complexity with redis connection handling (e.g., connection pooling), I'm thinking it's probably best to pass the redis instance this check should use. Leaving that complexity to the app keeps this check simpler.

Reply:
I've gone ahead and renamed to `redis_instance` which conveys what is being passed in via the parameters hash more clearly.